### PR TITLE
Fix #826 - Fixes unablity to select future disbursement dates

### LINKED ISSUE #826: Unable to select future dates for disbursement across all activities
**Summary:** 

Disbursement date is the date that the funds are actually made available to the borrower.It will be a future date.

**Steps to reproduce:** 

For instance.
1)Run the app
2)Add Loan account

**Expected behavior:** 

Able to select future dates in expected disbursement date.

**Observed behavior:** 

Not able to select dates beyond present.

**Device and Android version:** 

Android 5.0.2, API 21

**Screenshots:**

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
Fixes #826 

Please Add Screenshots If there are any UI changes.
video2gif_20191216_135641

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by OussEmaDevCode** [comment]
@luckyman20 @moksh-mahajan @santoshconflux @elirehema  Hi : ), can you please have a look at 
this !!!

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java [hunk 1]
```diff
@@ -44,6 +45,13 @@ public static MFDatePicker newInsance(Fragment fragment) {
         return mfDatePicker;
     }
 
+    public static MFDatePicker newInsance(Fragment fragment, boolean max) {
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java]:
```suggestion
    public static MFDatePicker newInstance(Fragment fragment, boolean max) {
```

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java [hunk 2]
```diff
@@ -33,6 +33,7 @@
     }
 
     OnDatePickListener onDatePickListener;
+    Boolean max = true;
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/uihelpers/MFDatePicker.java]: Can have better variable name.

**Comment by luckyman20** [comment]
@OussEmaDevCode Always keep commit title in this template - 'fix: <commit title>' for a bug fix.

**Comment by OussEmaDevCode** [comment]
@luckyman20 All requested changes have been done !!!

**Comment by Aditya-gupta99** [comment]
We are closing this PR as it causes a branch conflict since the Java file has now been migrated to kotlin

**Pull request closed without merging**